src/Tools/ExternalAccess/Razor/RazorAnalyzerAssemblyResolver.cs

```diff
@@ -19,32 +21,74 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.Razor
     [method: Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
     internal sealed class RazorAnalyzerAssemblyResolver() : IAnalyzerAssemblyResolver
     {
-        private const string RazorCompilerAssemblyName = "Microsoft.CodeAnalysis.Razor.Compiler";
-        private const string RazorUtilsAssemblyName = "Microsoft.AspNetCore.Razor.Utilities.Shared";
-        private const string ObjectPoolAssemblyName = "Microsoft.Extensions.ObjectPool";
+        internal const string RazorCompilerAssemblyName = "Microsoft.CodeAnalysis.Razor.Compiler";
+        internal const string RazorUtilsAssemblyName = "Microsoft.AspNetCore.Razor.Utilities.Shared";
+        internal const string ObjectPoolAssemblyName = "Microsoft.Extensions.ObjectPool";
```

Comment:
necessary to be internal?  nit: ide accessibilty approach is to having internal members be marked as 'public' (when already in an internal type).